docs/LOG_COLLECTION.md

```diff
@@ -0,0 +1,1205 @@
+
+```text
+/usr/share/nginx/html/
+└── <node-name>/
+    └── <timestamp>/
+        ├── nvidia-bug-report-<node-name>-<timestamp>.log.gz
+        ├── gpu-operator-must-gather-<node-name>-<timestamp>.tar.gz
+        ├── sosreport-<hostname>-<timestamp>.tar.xz  (if GCP SOS enabled)
+        └── sosreport-<hostname>-nvsentinel-<id>-<timestamp>.tar.xz  (if AWS SOS enabled)
+```
+
+**Example**:
+```text
+/usr/share/nginx/html/
+└── worker-node-01/
+    └── 20250106-143022/
+        ├── nvidia-bug-report-worker-node-01-20250106-143022.log.gz
+        └── gpu-operator-must-gather-worker-node-01-20250106-143022.tar.gz
+```
+
+---
+
+## When Logs Are Collected
+
+### Automatic Collection Triggers
+
+Logs are automatically collected when:
+
+1. **Fault Remediation Module detects a drain completion** on a node
+2. **Log collection is enabled** in the fault-remediation chart configuration
+3. **Node has experienced a fault** that triggered quarantine and drain
+
+### Configuration
+
+Enable log collection in your Helm values:
+
+```yaml
+faultRemediation:
+  enabled: true
+  logCollector:
+    enabled: true  # Set to true to enable automatic log collection
+    uploadURL: "http://nvsentinel-incluster-file-server.nvsentinel.svc.cluster.local/upload"
+    gpuOperatorNamespaces: "gpu-operator"  # Comma-separated list
+    enableGcpSosCollection: false  # Enable for GCP clusters
+    enableAwsSosCollection: false  # Enable for AWS clusters
+```
+
+### Job Lifecycle
+
+1. **Creation**: Fault-remediation module creates log collector job after node drain completes
+2. **Execution**: Job runs with privileged access on the target node
+3. **Collection**: Gathers all configured diagnostic logs (5-15 minutes typical duration)
+4. **Upload**: Uploads collected logs to file server
+5. **Completion**: Job completes and is cleaned up after TTL expires
+6. **TTL**: Job is automatically deleted 1 hour after completion (`ttlSecondsAfterFinished: 3600`)
+
+### Timeout Configuration
+
+```yaml
+logCollector:
+  collectionTimeout: 900  # 15 minutes default
+```
+
+---
+
+## How to Download Logs
+
+### Method 1: Port-Forward to File Server 
+
+This is the easiest way to browse and download logs from your local machine via web browser.
+
+#### Step 1: Set up port-forward
+
+```bash
+kubectl port-forward -n nvsentinel svc/nvsentinel-incluster-file-server 8080:80
+```
+
+#### Step 2: Access via web browser
+
+Open your browser to:
+```text
+http://localhost:8080
+```
+
+You'll see a directory listing with all node folders. Navigate to find your logs.
+
+#### Step 3: Download files
+
+Click on any file to download it directly from the browser.
+
+#### Alternative: Download via curl
+
+```bash
+# List all logs for a specific node
+curl http://localhost:8080/<node-name>/
+
+# Download a specific file
+curl -O http://localhost:8080/<node-name>/<timestamp>/nvidia-bug-report-<node-name>-<timestamp>.log.gz
+```
+
+### Method 2: Using dgxcops CLI (NVIDIA Internal Users)
```

Comment:
This should not be mentioned here, we should not mention anything that is used for internal users. Let's only keep option one.